release/ray_release/tests/test_config.py

```diff
@@ -14,8 +13,8 @@
 from ray_release.exception import ReleaseTestConfigError
 
 _TEST_COLLECTION_FILES = [
-    bazel_runfile("release/release_tests.yaml"),
-    bazel_runfile("release/ray_release/tests/test_collection_data.yaml"),
+    "release_tests.yaml",
+    "ray_release/tests/test_collection_data.yaml",
```

Comment:
why are you removing the `bazel_runfile`? could you keep it to be using bazel runfiles consistently?